src/components/dashboard/Dashboard.js

```diff
@@ -265,11 +268,28 @@ const Dashboard = props => {
     InteractionManager.runAfterInteractions(handleAppLinks)
   }
 
+  const saveBalanceBlockWidth = event => {
```

Comment:
please explain what you are doing here and why
also keep explanation as comment in code

Reply:
@sirpy this code needed for animation calculations. I need to have the width of the balance block to place it at the center as the balance always changes so the width is dynamical. Animation functionality requires the positioning props to be set with numbers. So I need to calculate the center within dynamically changed balance block width.

Reply:
I Added explanation comment to the code